### ISSUE TITLE
Add support for message position

### ISSUE THREAD

**Issue by hdsuperman** [DV8FromTheWorld/JDA#2305]
### General Troubleshooting

- [x] I have checked for similar issues on the Issue-tracker.
- [X] I have updated to the [latest JDA version](https://ci.dv8tion.net/job/JDA/lastSuccessfulBuild/)
- [X] I have checked the branches or the maintainers' PRs for upcoming features.

### Feature Request

https://discord.com/developers/docs/resources/channel#message-object

position: 
A generally increasing integer (there may be gaps or duplicates) that represents the approximate position of the message in a thread, it can be used to estimate the relative position of the message in a thread in company with total_message_sent on parent thread

### Example Use-Case

```java
var position message.getPosition();

if(position == 0) {
    log.info("Is first message");
}
```

**Comment by MinnDevelopment** [comment]
Can you provide a more concrete use-case of why you need this?

**Comment by freya022** [comment]
My use case would be to prevent people from using my bot by reacting on a message that's too old / too far away (the bot would only watch in threads anyway)

**Comment by MinnDevelopment** [comment]
Available in [5.0.0-beta.4](https://github.com/DV8FromTheWorld/JDA/releases/tag/v5.0.0-beta.4)
